udata/migrations/2023-02-08-rename-internal-dates.py

```diff
@@ -15,13 +16,23 @@ def migrate(db):
     dataset_collection = db.dataset
     dataset_collection.update_many({}, {'$rename': {
         'created_at': 'created_at_internal',
-        'last_modified': 'last_modified_internal',
-        'resource.created_at': 'resource.created_at_internal',
-        'resource.modified': 'resource.last_modified_internal'}})
+        'last_modified': 'last_modified_internal'}})
 
     community_resource_collection = db.community_resource
     community_resource_collection.update_many({}, {'$rename': {
         'created_at': 'created_at_internal',
-        'modified': 'last_modified_internal'}})
+        'modified': 'last_modified_internal',
+        'harvest.modified_at': 'harvest.last_modified'}})
+
+    datasets = Dataset.objects(resources__0__exists=True)
```

Comment:
`As mentioned in the documentation there is no way to directly rename fields within arrays with a single command. Your only option is to iterate over your collection documents, read them and update each with $unset old/$set new operations.`
According to this I did a loop for resources :/